### PR TITLE
Plugin Card: Update activa/inactive tag layout

### LINKED ISSUE #72340: Plugin Cards: Update the active/inactive tag layout

### PR DESCRIPTION
**Pull request by WBerredo**
#### Proposed Changes

* Update Installed icon and active/inactive tag layout
* Hide plugin ratings when there is active/inactive information to be shown

| Before  | After `active` | After `inactive`| After multi-site view|
| ------------- | ------------- |------------- |------------- |
|CleanShot 2023-01-23 at 11 07 20@2x|CleanShot 2023-01-24 at 11 34 34@2x|CleanShot 2023-01-24 at 11 32 22@2x|CleanShot 2023-01-24 at 11 32 42@2x|

#### Testing Instructions
* Install a plugin
* Search for its name on `/plugins`
* Check if the active version is similar to above (without ratings)
* Enter on the plugin page and deactivate it
* Go back to the search page and check if the inactive version is similar to above (without ratings)
* Select the multi-site view on the sidebar
* Check if the multi-site view is similar to above (with ratings)

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Fixes #72340

### CODE REVIEW

File: client/my-sites/plugins/plugins-browser-item/index.jsx [hunk 1]
```diff
@@ -251,23 +264,16 @@ const InstalledInOrPricing = ( {
...
-		if ( selectedSiteId ) {
-			checkmarkColorClass = isPluginActiveOnsiteWithSubscription
-				? 'checkmark--active'
-				: 'checkmark--inactive';
-		}
 		return (
 			<div className="plugins-browser-item__installed-and-active-container">
 				<div className="plugins-browser-item__installed ">
-					<Gridicon icon="checkmark" className={ checkmarkColorClass } size={ 14 } />
+					<Gridicon icon="checkmark" className="checkmark" size={ 12 } />
```
Comment (gcsecsey) [inline client/my-sites/plugins/plugins-browser-item/index.jsx:285]: Active and inactive tags work great for me, multisite badges are shown for me with a grey checkmark (compared to the green one in the PR description):
CleanShot 2023-01-24 at 13 31 19@2x

I guess this is beacuse the class names have been changed. Is this OK if the checkmark is grey? IMO it looks great! 👍

Comment (WBerredo) [reply client/my-sites/plugins/plugins-browser-item/index.jsx:285]: Yes, it was an intentional change, as you can see the checkmark is shown as gray on the layouts (pdh6GB-2At-p2).
The green checkmark is displayed as a `Before` version on the PR description. 

Let me know if I understood anything wrong.

**Review by JulianBustamante — APPROVED** [review]
LGTM!

Active
Screenshot 2023-01-24 at 2 28 53 PM

After Inactive
Screenshot 2023-01-24 at 2 30 05 PM

Multi-site view
Screenshot 2023-01-24 at 2 31 16 PM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7770189

Thank you @WBerredo for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7770189).

**Pull request merged**